### PR TITLE
add Governance for the Standards Group

### PR DESCRIPTION
**Pull request by sendilkumarn**
r? @MylesBorins 

Based on the discussion from #21 #19

### CODE REVIEW

**Comment by sendilkumarn** [comment]
Is there something pending to do here?

File: GOVERNANCE.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+If an agenda item cannot reach a consensus, a Team member can call for
+the item to be decided by a vote or to table the issue to the next
+meeting. In both cases the decision must be seconded by a majority of the Team
+or else the discussion will continue. Simple majority wins. Only Active
+Members participate in a vote.
+
+## Pull Requests
+
+This section details expectations for team members involved in any
+pull requests that relate to the group's scope.
```
Comment (MylesBorins) [inline GOVERNANCE.md]: I think the above lines can be removed

File: GOVERNANCE.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+Pull requests are merged if it goes unopposed and at least two members approve it within 72 hours after creating.
```
Comment (MylesBorins) [inline GOVERNANCE.md]:
```suggestion
Pull requests require approval by at least two members and must remain open for at least 72 hours before being landed. If there are any objections the Pull Request cannot land until they are resolved.
```

File: GOVERNANCE.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+the item to be decided by a vote or to table the issue to the next
+meeting. In both cases the decision must be seconded by a majority of the Team
+or else the discussion will continue. Simple majority wins. Only Active
+Members participate in a vote.
+
+## Pull Requests
+
+Pull requests require approval by at least two members and must remain open for at least 72 hours before being landed. If there are any objections the Pull Request cannot land until they are resolved.
+
+A pull request that is opposed will be discussed in the meeting.
```
Comment (MylesBorins) [inline GOVERNANCE.md]: I think this can be inferred from the above paragraph.... this is a tiny nit though

**Comment by jorydotcom** [comment]
I really think this is a great first version of this doc, thank you for doing this work @sendilkumarn. We did discuss it at the meeting yesterday, but of course want to make sure everyone has had a chance to see it an +1. 

For avoidance of doubt, can we get the @openjs-foundation/standards team & others to add their +1 / review approval?

File: GOVERNANCE.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+The following groups automatically qualify for membership and can request to be added to the GitHub team:
+
+* OpenJS Foundation CPC Members
+* OpenJS Foundation Project Maintainers
+
+## Team Meetings
+
+The Team meets bi-weekly on Zoom.us. A designated moderator
+approved by the Team runs the meeting. Each meeting should be
+published to YouTube.
```
Comment (Eomm) [inline GOVERNANCE.md]: If the streaming doesn't work for any reasons, should the meeting be postponed?
It happened to one node.js meeting and to be 100% transparency we did it so

**Comment by MylesBorins** [comment]
The meeting can be recorded and posted afterwards

**Pull request merged**